Automattic/simplenote-ios
# Fix History Crash on iOS 12

#209 appears to be caused by the usage of `MagnifierView` in the history browser, which magnifies the text of the horizontal picker for the item in the center view. MagnifierView was copied from [this blog post](https://coffeeshopped.com/2010/03/a-simpler-magnifying-glass-loupe-view-for-the-iphone). I believe the issue is related to calling `renderInContext` from `drawRect` which now created an infinite loop on iOS 12. I couldn't figure out how to fix it, so have removed `MagnifierView` entirely in this PR as the difference in the UI is negligible. 

**To Test**
* Open a note that has history
* Tap on `i` button, and then `History`.
* You should be able to browse the history without a crash on iOS 12.

Fixes #209
